packages/apostrophe-widgets/share-links/src/share-links.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+import "./share-links.css";
+
+import "@utrecht/component-library-css";
+import "@utrecht/design-tokens/dist/root.css";
+import { LinkSocial, Heading4 } from "@utrecht/component-library-react";
+
+interface Item {
+  title: string;
+}
+
+function ShareLinks({ title }: Item) {
+  return (
+    <div className="share-links">
+      <Heading4>{title}</Heading4>
+      <LinkSocial
+        external
+        href={'http://www.facebook.com/share.php?u=' + encodeURIComponent(location.href)}
+        title={'Facebook'}
+      >
+        <i className='icon --facebook'></i>
+      </LinkSocial>
+
+      <LinkSocial
+        external
+        href={'https://twitter.com/intent/tweet?text=' + encodeURIComponent(location.href)}
+        title={'Twitter'}
+      >
+        <i className='icon --twitter'></i>
+      </LinkSocial>
+
+      <LinkSocial
+        external
+        href={'mailto:?subject='+document.title+'&body=' + encodeURIComponent(location.href)}
+        title={'Mail'}
+      >
+        <i className='icon --mail'></i>
+      </LinkSocial>
+
+      <LinkSocial
+        external
+        href={'https://api.whatsapp.com/send?phone=&text='+ encodeURIComponent(location.href)+'&source=&data='}
+        title={'Whatsapp'}
+      >
+        <i className='icon --whatsapp'></i>
+      </LinkSocial>
```

Comment:
Voor nu prima, maar zou mooi zijn als het instelbaar is welke socials je wilt laten zien. Volgorde mag vast staan, maar gewoon een yes/no per social.